virtualmachine/aws/wait.go

```diff
@@ -9,6 +9,20 @@ import (
 	"github.com/aws/aws-sdk-go/service/ec2"
 )
 
+const (
```

Comment:
If these constants are not cloud specific, should we move them to ```virtulamachine/.....``` ?

Reply:
Checked in GCP and VSphere cloud, that wait functions are differently implemented in different clouds. For ex in VSphere I couldn't find any timeout used, in GCP there is wait function implemented but timeout is set to 180 seconds only.
So considering that some clouds may have wait functions in-built with the operation call, or difference in timeout and retry duration required across clouds, its better to specify it cloud specific only.